packages/core/src/expression-builders/identifier.ts

```diff
@@ -28,6 +31,6 @@ export class Identifier extends BaseSqlExpression {
  * SELECT * FROM users WHERE "firstName" = 'John'
  * ```
  */
-export function identifier(value: string): Identifier {
-  return new Identifier(value);
+export function identifier(...values: Array<string | TableOrModel>): Identifier {
```

Comment:
Is there a reason we might want to allow multiple `TableOrModel` here?

Reply:
Not really, but it was easier to type

The thing I was hoping to simplify by allowing multiple values is something like `sql.identifier('databaseName', Model, 'columnName')` but as I don't exactly know where the Model will be used (as databaseName and columnName is optional), I just allowed it everywhere

Reply:
Oh, that's a use case I didn't think of. It might be good to add a test for that as well so we know that something like that is supported. I was mainly thinking about things like `sql.identifier(Model1, Model2)` which don't make sense, but that's difficult to type

Reply:
Here we go!